serializr.js

```diff
@@ -1,7 +1,9 @@
 (function(g) {
     "use strict"
-
+    
     function mrFactory() {
+        // Indicate properties should be skipped
+        var SKIP = Symbol('SKIP')
```

Comment:
Note that Symbol is not standard ES5, so something like `typeof Symbol !== "undefined" ? Symbol('SKIP') : { SKIP: true }` would be better. I'll try to merge later today, so I might process that change myself